packages/create-react-app/index.js

```diff
@@ -111,20 +147,22 @@ function createApp(name, verbose, version, template) {
   checkAppName(appName);
   fs.ensureDirSync(name);
   if (!isSafeToCreateProjectIn(root)) {
-    console.log('The directory ' + chalk.green(name) + ' contains files that could conflict.');
+    console.log(
+      'The directory ' +
+        chalk.green(name) +
+        ' contains files that could conflict.'
```

Comment:
What about this one? @vjeux

I was a bit surprised it broke lines on every concat here (even if it fits). Maybe it's needed for uniformity with other cases but it's a bit annoying for concatenating strings with data.